docker/Dockerfile.build-linux

```diff
@@ -18,6 +18,10 @@ RUN apt-get update \
 RUN update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-8 30 \
   && update-alternatives --install /usr/bin/clang clang /usr/bin/clang-8 30
 
+RUN GRPC_HEALTH_PROBE_VERSION=v0.3.6 && \
```

Comment:
This CLI tool can be used in k8s environments for liveness and startup probes instead of vanilla HTTP probes which won't work right out of the box for a gRPC server. See livepeer/livepeer-infra#416.